tasks/src/main/scala/dagr/tasks/picard/SamToFastq.scala

```diff
@@ -46,17 +46,26 @@ class SamToFastq(in: PathToBam,
                  fastq2: Option[PathToFastq] = None,
                  interleave : Boolean = true,
                  clippingAttribute: Option[String] = None,
-                 clippingAction: Option[String] = None) extends PicardTask with Pipe[SamOrBam,Fastq] {
+                 clippingAction: Option[String] = None,
+                 clippingMinReadLength: Int = 25) extends PicardTask with Pipe[SamOrBam,Fastq] {
```

Comment:
It may be overkill, but having `clippingMinReadLength` be an option may make it clearer that attribute/action/minreadlength are set together.